CHANGELOG.md

```diff
@@ -24,7 +24,7 @@
 - Decreased release binary size by ~15%.
 - Changed default API socket path to `/run/firecracker.socket`. This path
   also applies when running with the jailer.
-- Disabled KVM dirty page tracking by default.
+- Enabled KVM dirty page tracking by default.
```

Comment:
I think the title of the PR would make a better changelog entry than this one. That `by default` is rather confusing.

Reply:
I didn't mention the new functionality in the changelog because without an associated api it's not user facing; what if I remove the `by default`? I think the changelog should still mention that a flag previously disabled is now enabled.

Reply:
What about the classical `Support for KVM dirty page tracking`?